src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -694,6 +694,31 @@ static int _write_script_header_to_fd(struct xccdf_policy *policy, struct xccdf_
 		const struct xccdf_version_info *xccdf_version = xccdf_benchmark_get_schema_version(benchmark);
 		const char *xccdf_version_name = xccdf_version_info_get_version(xccdf_version);
 
+		if (NULL != profile_description) {
+			unsigned int newLines = 0;
+			const char *c = profile_description;			
+			while (*c != '\0') {
+				if (*c == '\n')
+					++newLines;
+				++c;
+			}
+			
+			if (newLines > 0) {
+				unsigned int description_length = c - profile_description;
+				const char filler[] = "#                       ";
+				char *commented_description = malloc(description_length + newLines * (sizeof filler - 1));
+				for (unsigned int i = 0, j = 0; j < description_length; ++i, ++j) {
+					commented_description[i] = profile_description[j];
+					if (profile_description[j] == '\n') {
+						memcpy(&commented_description[i + 1], filler, (sizeof filler - 1));
```

Comment:
This is undefined behavior. It sucks but that's what it is. I recommend looking into `memcpy` vs `memmove`.

Reply:
you also probably want `strcpy` here